src/test/java/me/karakelley/tictactoe/EasyBehaviorTest.java

```diff
@@ -0,0 +1,19 @@
+package me.karakelley.tictactoe;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.spy;
+
+class EasyBehaviorTest {
+
+  BoardState boardState = spy(new BoardState());
+
+  @Test
```

Comment:
you could write more specific tests. Like set up the boardState with some occupied and available cells and assert the pickedMove is included in the available cells